integration_testing/features/cancelling-a-sync-operation.feature

```diff
@@ -0,0 +1,13 @@
+Feature:
+
+	Background: Cancelling a sync operation

```

Comment:
We prefer to use indicative/imperative mode (`Cancel`) to continuous (`Cancelling`) in UX writing (procedural steps descriptions in Gherkin scenarios and general technical writing.